website/docs/customize-admission.md

```diff
@@ -21,16 +21,19 @@ configuration options deserve special mention: [timeouts](https://kubernetes.io/
 
 Timeouts allow you to configure how long the API server will wait for a response from the admission webhook before it
 considers the request to have failed. Note that setting the timeout longer than the overall request timeout
-means that the main request will time out before the webhook's failure policy is invoked.
+means that the main request will time out before the webhook's failure policy is invoked, causing the
+request to fail.
 
 Failure policy controls what happens when a webhook fails for whatever reason. Common
 failure scenarios include timeouts, a 5xx error from the server or the webhook being unavailable.
 You have the option to ignore errors, allowing the request through, or failing, rejecting the request.
 This results in a direct tradeoff between availability and enforcement.
 
-Currently Gatekeeper is defaulting to using `Ignore` for the constraint requests. This is because
-the webhook server currently only has one instance, which risks downtime during actions like upgrades.
-As the theoretical availability improves we will likely change the default to `Fail`.
+Currently Gatekeeper is defaulting to using `Ignore` for the constraint requests, which means
```

Comment:
Should we mention with the exception of encountering Rego syntax error during runtime? ref: https://github.com/open-policy-agent/gatekeeper/issues/1562

Reply:
Good question. An alternative short-term would be to send a 5XX error on Rego runtime errors, then the default webhook failure behavior would apply... this might be more in line with user expectations.

Long term, for #1562 we should isolate Rego runtime errors.

WDYT?

Reply:
sgtm

Reply:
which approach SGTY? Adjusting our failure response on Rego runtime errors?

Reply:
Both sgtm, but we can start with send a 5XX error on Rego runtime errors as short term.

Reply:
It looks like we do send 5XX errors, but maybe the API server doesn't interpret those as expected?

It looks like controller-runtime added some pre-baked response functions. Switching to those in the hopes that they'll be better-behaved:

#1581

Merging this PR in the meantime.